config/initializers/sucker_punch.rb

```diff
@@ -0,0 +1 @@
+require 'sucker_punch/async_syntax'
```

Comment:
This is needed until Rails 5.0, when the queue adapter is changed to async_perform from new.async.perform.  Could add a comment in the initializer that it can be removed when Rails is 5.0 or newer.